releasenotes/notes/agent-tel-extend-histogr-6e2da94e63edcaf8.yaml

```
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Extended Agent telemetry histogram details, specifically:
+      - Added to a histogram's payload previously omitted and implicit `+Inf` bucket value
```

```suggestion
      - Added previously omitted and implicit `+Inf` bucket value to histogram payload 
```
Not sure if this changes the meaning. I found the original sentence a little hard to parse